lib/stub_ui/lib/src/ui/text.dart

```diff
@@ -1106,6 +1106,39 @@ enum BoxWidthStyle {
   max,
 }
 
+/// Where to vertically align the placeholder relative to the surrounding text.
+/// Used by [ParagraphBuilder.addPlaceholder].
```

Comment:
nit: here and everywhere else: The first paragraph should be a one-sentence summary: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph